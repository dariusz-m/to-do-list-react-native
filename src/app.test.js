import React from 'react';
import renderer from 'react-test-renderer';
import ToDoApp from './app';

it('Test whole the app can be rendered', () => {
    const tree = renderer.create(
        <ToDoApp/>,
    ).toJSON();

    expect(tree).toMatchSnapshot();
});
