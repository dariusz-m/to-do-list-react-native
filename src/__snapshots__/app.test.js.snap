// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test whole the app can be rendered 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 40,
          "fontWeight": "bold",
        }
      }
    >
      Sign In
    </Text>
    <View>
      <Text
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
        Username: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={true}
        onChangeText={[Function]}
        placeholder="username..."
        secureTextEntry={false}
        style={
          Object {
            "borderColor": "black",
            "borderWidth": 0.2,
            "height": 40,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
    <View>
      <Text
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
        Password: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={true}
        onChangeText={[Function]}
        placeholder="password..."
        secureTextEntry={true}
        style={
          Object {
            "borderColor": "black",
            "borderWidth": 0.2,
            "height": 40,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
    <View
      accessibilityComponentType="button"
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
              Object {
                "color": "blue",
              },
            ]
          }
        >
          Sign in
        </Text>
      </View>
    </View>
  </View>
</View>
`;
